app/models/ext_management_system.rb

```diff
@@ -269,6 +269,7 @@ def validate_zone_not_maintenance_when_ems_enabled?
   virtual_column :supports_cinder_volume_types, :type => :boolean
   virtual_column :supports_volume_availability_zones, :type => :boolean
   virtual_column :supports_create_security_group, :type => :boolean
+  virtual_column :supports_storage_services, :type => :boolean
```

Comment:
@agrare Note to ourselves... This pattern is weird... I don't like that we expose these as virtual columns, because that, in turn, exposes them to reporting.  I get we need them over the API, but I'm wondering if we do this differently.  Additionally, down below, we seem to always need to add the `?`-less method, but we can just do that directly in support_feature_mixin, right?

Reply:
Yeah we could create an alias automatically, I think it is really only used by `virtual_column` though so if change that we might not need to build those?